GenericNode/src/main/java/genericnode/Clients/TCPClient.java

```diff
@@ -0,0 +1,71 @@
+package genericnode.Clients;
+
+import java.io.DataInputStream;
+import java.io.DataOutputStream;
+import java.io.IOException;
+import java.net.Socket;
+import java.util.AbstractMap.SimpleEntry;
+
+public class TCPClient extends Client {
+
+    Socket clientSocket;
+    DataOutputStream outToServer = null;
+    DataInputStream inFromServer = null;
+    @Override
+    public void put(String key, String value) throws IOException {
+        outToServer.writeUTF(key);
+        outToServer.writeUTF(value);
+        clientSocket.close();
+    }
+
+    @Override
+    public String get(String key) throws IOException {
+        outToServer.writeUTF(key);
+        System.out.println(inFromServer.readUTF());
+        return null;
+    }
+
+    @Override
+    public void del(String key) throws IOException {
+        outToServer.writeUTF(key);
```

Comment:
put command should output the result like:
$ java -jar GenericNode.jar tc localhost 1234 del a
server response:delete key=a